lib/dash/content_protection.js

```diff
@@ -320,6 +320,50 @@ shaka.dash.ContentProtection = class {
     return ContentProtection.getLaurl_(rootElement);
   }
 
+
+  /**
+   * Gets a PlayReady initData from a content protection element
+   * containing a PlayReady Pro Object
+   *
+   * @param {shaka.dash.ContentProtection.Element} element
+   * @return {?Array.<shaka.extern.InitDataOverride>}
+   */
+  static getInitDataFromPro(element) {
```

Comment:
We should probably make this a general utility method.  I suggest moving the bulk of this to the `shaka.util.Pssh` type as a static method.  It would accept a system ID and data buffers and create the `pssh` blob from it.